charts/flux2/values.yaml

```diff
@@ -275,3 +275,10 @@ extraObjects: []
 prometheus:
   podMonitor:
     create: false
+    podMetricsEndpoints:
+      - port: http-prom
+        relabelings:
+          # https://github.com/prometheus-operator/prometheus-operator/issues/4816
```

Comment:
@acondrat can you please open a PR and update the podMonitor in the Flux repo? https://github.com/fluxcd/flux2/tree/main/manifests/monitoring/monitoring-config